std/lua/_std/Reflect.hx

```diff
@@ -121,6 +121,15 @@ import lua.Boot;
 				|| (t == "function" && (lua.Boot.isClass(v) || lua.Boot.isEnum(v)) != null);
 		}
 
+	public static function isStructure(v:Dynamic):Bool untyped {
+		if (v == null) return false;
+		if(Lua.type(v) != "table" || v.__enum__ != null || lua.Boot.isClass(v) || lua.Boot.isEnum(v)) return false;
+		var mt = Lua.getmetatable(v);
+		if(mt == null) return true;
+		if(mt.__index != null) return false;
+		return !Reflect.hasField(mt, '__index');
```

Comment:
this is needed because `Type.createEmptyInstance(ReallyEmptyClass)` __index is null